src/stores/launchdarkly.ts

```diff
@@ -14,6 +12,9 @@ export const useBcrosLaunchdarkly = defineStore('bcros/launchdarkly', () => {
   const ldInitialized = ref(false)
 
   function init () {
+    const keycloak = useBcrosKeycloak()
+    const account = useBcrosAccount()
```

Comment:
I moved these in here so they're evaluated when init is called. Otherwise, `account` is called before account is set and the result is undefined, meaning we could never target by org (line 41 below).